examples/magic_image_arithmetic.py

```diff
@@ -47,8 +47,8 @@ def image_arithmetic(
 
 # create a new viewer with a couple image layers
 viewer = napari.Viewer()
-viewer.add_image(np.random.rand(20, 20), name='Layer 1')
-viewer.add_image(np.random.rand(20, 20), name='Layer 2')
+viewer.add_image(np.random.rand(20, 20), name='Layer 1', scale=(2, 2))
```

Comment:
This change isn't really making the example more meaningful without a comment. I would add a comment here and say that we will inherit this scale value in the return value of the widget.